dashboards/Mesos-Masters-Summary.json

```diff
@@ -938,7 +940,15 @@
       },
       "lines": true,
       "linewidth": 1,
-      "links": [],
+      "links": [
+        {
+          "dashboard": "Mesos: Frameworks Summary",
+          "keepTime": true,
+          "title": "Mesos: Frameworks Summary",
+          "type": "dashboard",
+          "url": "/service/dcos-monitoring/grafana/d/C3EFGTeiz/mesos-frameworks-summary"
```

Comment:
unfortunately this url can change depending on what name the service is installed with (it defaults to `dcos-monitoring` but can be set to something else e.g. in SOAK it is configured to `monitoring` which means this url would change to `/service/monitoring/grafana...`

Reply:
This seems bizarre - that is a Grafana feature rendering this URL.

Reply:
@gracedo I have now switched those URLs to be relative, hoping that will do the job.

We should have similar problems with URLs like this, no?
https://github.com/dcos/grafana-dashboards/blob/master/dashboards/AdminRouter/AdminRouter-Details-DCOS-Component-Proxy-Responses.json#L187

Reply:
Yes, unfortunately those will also be broken if installed with a different service name. As a matter of fact, I think I saw in a tech enablement demo today that the admin router drill down links in mwt grafana were broken (I imagine due to this reason)

Reply:
Damnit - thanks for reporting this - hopefully relative URLs do the trick.

Reply:
Do the relative URLs work? If you loaded these dashboards on a cluster and the links work, I'll +1 this :)

Reply:
YEah - I found that trick earlier when trying to use markdown links to other dashboards - figured the service name was a problem. I totally ignored the fact that grafana itself was rendering such absolute URLs for the "drill down" link feature, somewhat assuming some magic would fix that.